enabler/src/de/schildbach/pte/FranceNorthEastProvider.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.pte;
+
+import de.schildbach.pte.dto.Product;
+import de.schildbach.pte.dto.Style;
+import de.schildbach.pte.dto.Style.Shape;
+
+/**
+ * @author Stéphane Guillou
+ */
+public class FranceNorthEastProvider extends AbstractNavitiaProvider
+{
+	private static String API_REGION = "fr-ne";
+
+	public FranceNorthEastProvider(final String apiBase, final String authorization)
+	{
+		super(NetworkId.FRANCENORTHEAST, apiBase, authorization);
```

Comment:
Did you try to compile your code? `NetworkId.FRANCENORTHEAST` does not seem to be added by your PR, so it should fail to compile.